pyrit/score/scorer.py

```diff
@@ -130,3 +133,69 @@ def get_identifier(self):
         identifier["__module__"] = self.__class__.__module__
         identifier["sub_identifier"] = None
         return identifier
+
+    @pyrit_json_retry
+    async def send_chat_target_async(
+        self,
+        *,
+        prompt_target: PromptChatTarget,
+        scorer_llm_request: PromptRequestResponse,
+        scored_prompt_id: str,
+        category: str = "",
+        task: str = "",
+    ) -> UnvalidatedScore:
+        """
+        Sends a request to a target LLM, and takes care of retries.
```

Comment:
```suggestion
        Sends a request to a target model, and takes care of retries.
```
Here and elsewhere. Or perhaps "endpoint" to be more generic?